lib/hanami.rb

```diff
@@ -118,6 +118,33 @@ def self.app
     App.new(configuration, environment)
   end
 
+  # Check if an application is allowed to load.
+  #
+  # The list of applications to be loaded can be set via the `HANAMI_APPS`
+  # env variable. If the HANAMI_APPS env variable is not set, it defaults
+  # to loading all applications.
+  #
+  # @return [TrueClass,FalseClass] the result of the check
+  #
+  # @since x.x.x
+  #
+  # @example
+  #
+  #   # Mount hanami app for specific app
+  #   Hanami.configure do
+  #     if Hanami.app?(:web)
+  #       require_relative '../apps/web/application'
+  #       mount Web::Application, at: '/'
+  #     end
+  #   end
+  #
+  def self.app?(app)
+    return true unless ENV.key?('HANAMI_APPS')
+
+    allowed_apps = ENV['HANAMI_APPS'].to_s.split(',')
+    allowed_apps.include?(app.to_s.downcase)
```

Comment:
@davydovanton You're right, I got confused by `downcase`.

One thing we should still make clear. if the code won't return `true` for this:

```ruby
Hanami.app?("CamelCaseApp") # => false
Hanami.app?("camel_case_app") # => true
```

Where is gonna to be defined this rule? Do we want to document this? 😉